### DISCUSSION TITLE
GCP Managed Postgres support

### DISCUSSION THREAD

**Discussion by jschaf** [cybertec-postgresql/pg_timetable#447] [General]
I’m interested in migrating from pgcron. The readme shows that GCP is supported but pg_timetable isn’t listed as one of the supported extensions at  https://cloud.google.com/sql/docs/postgres/extensions#postgresql-extensions-supported-by-cloud-sql.

Is the extension supported on GCP?

**Comment by pashagolub** [comment]
The thing is that `pg_timetable` is not an extension. It's a standalone application you may run wherever you want. And yes, GCP is fully supported

**Reply by jschaf** [reply]
Ah, thank you. That makes more sense. I think I was biased by pg_cron being an extension.
